scripts/generate-migration-tvm.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import csv
+import sys
+
+def generateRollingStockTvmCompatibilitySql(filename):
+  with open(filename, newline='') as csvfile:
+    reader = csv.DictReader(csvfile)
+    whereClause300 = "WHERE name IN ("
+    whereClause430 = "WHERE name IN ("
+    whereClauseBoth = "WHERE name IN ("
+    for row in reader:
+      rowCodeEngin = row['Code Engin  [Ref]']
+      rowTvm300 = row['MR équipé TVM 300 [O, N]']
+      rowTvm430 = row['MR équipé TVM 430 [O, N]']
+      if rowTvm300 == 'O' and rowTvm430 == 'N':
+        if len(whereClause300) != 15:
+          whereClause300 += ","
+        whereClause300 += f"'{rowCodeEngin}'"
+      if rowTvm300 == 'N' and rowTvm430 == 'O':
+        if len(whereClause430) != 15:
+          whereClause430 += ","
+        whereClause430 += f"'{rowCodeEngin}'"
+      if rowTvm300 == 'O' and rowTvm430 == 'O':
+        if len(whereClauseBoth) != 15:
+          whereClauseBoth += ","
+        whereClauseBoth += f"'{rowCodeEngin}'"
+
+    whereClause300 += ");"
+    whereClause430 += ");"
+    whereClauseBoth += ");"
+    print("UPDATE rolling_stock")
+    print("SET supported_signaling_systems='[\"BAPR\", \"BAL\", \"TVM300\"]'")
+    print(whereClause300)
+    print("")
+
+    print("UPDATE rolling_stock")
+    print("SET supported_signaling_systems='[\"BAPR\", \"BAL\", \"TVM430\"]'")
+    print(whereClause430)
+    print("")
+
+    print("UPDATE rolling_stock")
+    print("SET supported_signaling_systems='[\"BAPR\", \"BAL\", \"TVM300\", \"TVM430\"]'")
+    print(whereClauseBoth)
+    print("")
+
+def main():
+    if len(sys.argv) > 1:
+        input_string = sys.argv[1]
+        generateRollingStockTvmCompatibilitySql(input_string)
+    else:
+        print("Please provide csv filename")
+
+if __name__ == "__main__":
+    main()
```

Comment:
This script does not belong here:
- nobody will run it ever again
- it only works on proprietary data

Reply:
removed